src/ignitus-JobConfirmation/Components/index.js

```diff
@@ -0,0 +1 @@
+export { default as JobConfirmation } from './JobConfirmation';
```

Comment:
`Codacy` complains about it, please have a look.

Reply:
But every page has same kind of export. What is alternative of this?

Reply:
`export default JobConfirmation from './JobConfirmation';`
But for code consistency lets leave it as it is. :)